docs/src/main/paradox/docs/getting-started/running-nexus/configuration/index.md

```diff
@@ -0,0 +1,91 @@
+# Nexus configuration
+
+Nexus Delta service can be highly customized using @link:[configuration file(s)](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/resources/app.conf){ open=new }. Many things can be adapted to your deployment needs: port where the service is running, timeouts, the database you decide to support, pagination defaults, etc. 
+
+There are 2 ways to modify the default configuration:
+
+- Using JVM properties as arguments when running the service: -D`{property}`. For example: `-Dapp.http.interface="127.0.0.1"`.
+- Using @link:[FORCE_CONFIG_{property}](https://github.com/lightbend/config#user-content-optional-system-or-env-variable-overrides){ open=new }
+  environment variables. In order to enable this style of configuration, the JVM property
+  `-Dconfig.override_with_env_vars=true` needs to be set. Once set, a configuration flag can be overridden. For example: `CONFIG_FORCE_app_http_interface="127.0.0.1"`.
+  
+In order to successfully run Nexus Delta there is a minimum set of configuration flags that need to be specified
+
+
+## Http configuration
+
+@link:[This section](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/resources/app.conf#L9){ open=new } of the configuration defines the binding address and port where the service will be listening.
+
+## Database configuration
+
+Since 1.5.0 Nexus Delta comes in two database flavours: postgres or cassandra. The configuration flag `app.database.flavour` is used to select the flavour.
+
+## Cassandra configuration
+
+@link:[This section](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/resources/app.conf#L58){ open=new } of the configuration defines the cassandra specific configuration (username, password, contact points, etc).
```

Comment:
We should also have a link to the application-cassandra.conf
We should emphasize that first-time-bucket is an important setting